custom_components/tuya_local/devices/yym_805SW_aroma_nightlight.yaml

```diff
@@ -5,19 +5,36 @@ products:
     name: YYM-805SW
   - id: 4870500398f4abbfbae4
     name: GX
+  - id: 58107061e868e7c462da
+    name: Tellur WiFi Aroma Diffuser
 primary_entity:
   entity: fan
   dps:
     - id: 1
       name: switch
       type: boolean
       mapping:
-        - dps_val: true
-          icon: mdi:scent
-          value: ON
-        - dps_val: false
-          icon: mdi:scent-off
-          value: OFF
+        - conditions:
```

Comment:
true, on and ON are all the same value in yaml, likewise false, off and OFF.   This change just complicates the config, especially with the added condition that applies to both true and false cases.

Reply:
Yeah, I was hoping I could address this bug: https://github.com/make-all/tuya-local/issues/936

But then I did more research and I realized the mappings are fine